generic_expression_patterns_modules/process.py

```diff
@@ -289,7 +291,10 @@ def generate_summary_table(
         }
     )
     summary["Z score"] = (
-        summary["Test statistic (Real)"] - summary["Mean test statistic (simulated)"]
+        abs(
```

Comment:
Would it be useful to have a column in the summary table for the z-score, and then another one for the absolute value of the z-score? Or do you only expect to use the absolute value?

If the latter, I'd suggest naming the column something like `abs(Z score)`, just to remind yourself later on that these aren't the untransformed z-scores.

Reply:
We are using z-score to determine the "specificity" of genes so we care if they are different then the null, but I don't think we care about the direction for this. So I will add this text to the header I think that will definitely help future me!